scripts/ethereum-dev-node.ts

```diff
@@ -0,0 +1,81 @@
+#!/usr/bin/env -S yarn node -r ts-node/register/transpile-only
```

Comment:
Cool trick! Is it better than having a `yarn` command?

Reply:
I prefer this because you can also execute the script directly from the terminal. Curiously this is also faster to start up than having a yarn script. Finally, we don’t need to maintain a script entry for script and this avoid breaking things when scripts are moved.